source/plugin/Assets/GoogleMobileAds/Common/DummyClient.cs

```diff
@@ -128,31 +151,70 @@ public int GetDeviceSafeWidth()
         public void CreateBannerView(string adUnitId, AdSize adSize, AdPosition position)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor) {
+                dummyAd = Resources.Load(prefabAds[adSize]) as GameObject;
+                AdBehaviour.AnchorAd(dummyAd,position);
+            }
         }
 
         public void CreateBannerView(string adUnitId, AdSize adSize, int positionX, int positionY)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor)
+            { 
+                dummyAd = Resources.Load(prefabAds[adSize]) as GameObject;
+                Image myImage = dummyAd.GetComponentInChildren<Image>();
+                RectTransform rect = myImage.GetComponent<RectTransform>();
+                rect.anchoredPosition = new Vector3(positionX, positionY, 1);
+            }
         }
 
         public void LoadAd(AdRequest request)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor)
+            {
+                Task.Delay(1000).ContinueWith(_ => MobileAdsEventExecutor.ExecuteInUpdate(ShowBannerView));
+            }
         }
 
         public void ShowBannerView()
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor)
+            {
+                Image myImage = dummyAd.GetComponentInChildren<Image>();
+                RectTransform rect = myImage.GetComponent<RectTransform>();
+                AdBehaviour.ShowAd(dummyAd, rect.anchoredPosition);
+                isOpen = true;
+            }
         }
 
         public void HideBannerView()
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor && isOpen)
+            {
+                AdBehaviour.DestroyAd(dummyAd);
+                isOpen = false;
+            }
         }
 
         public void DestroyBannerView()
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+            if (inUnityEditor && isOpen)
```

Comment:
is this needed? Can't we just do a null check?

Reply:
Oh yeah! the isOpen is definitely not necessary I'll change that